src/main/kotlin/io.javalin/HaltException.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2017 David Åse
+ * Licensed under Apache 2.0: https://github.com/tipsy/javalin/blob/master/LICENSE
+ */
+
+package io.javalin
+
+import javax.servlet.http.HttpServletResponse
+
+class HaltException : RuntimeException {
+    var statusCode = HttpServletResponse.SC_OK
+    var body = "Execution halted"
+
+    constructor() {}
+
+    constructor(statusCode: Int) {
```

Comment:
I couldn't get that to work. Since it's being called with single arguments from Java, I needed to do 
```
class HaltException(var statusCode : Int = 200, var body : String = "Execution halted") : RuntimeException() {
    constructor(statusCode: Int) : this() {
        this.statusCode = statusCode;
    }
    constructor(body: String) : this() {
        this.body = body;
    }
}
```
Unless I missed something?